src/sentry/feedback/endpoints/feedback_ingest.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import annotations
+
+import datetime
+from typing import Any, Dict
+
+from rest_framework import serializers
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.authentication import (
+    ApiKeyAuthentication,
+    DSNAuthentication,
+    OrgAuthTokenAuthentication,
+    TokenAuthentication,
+)
+from sentry.api.base import Endpoint, region_silo_endpoint
+from sentry.api.bases.project import ProjectPermission
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.constants import ObjectStatus
+from sentry.feedback.models import Feedback
+from sentry.models import Organization, ProjectKey
+from sentry.models.project import Project
+from sentry.utils.sdk import bind_organization_context, configure_scope
+
+
+class FeedbackValidator(serializers.Serializer):
+    contexts = serializers.JSONField(required=False)
+    tags = serializers.JSONField(required=False)
+    timestamp = serializers.FloatField(required=False)
+    transaction = serializers.CharField(required=False)
+    type = serializers.CharField(required=False)
+    transaction_info = serializers.JSONField(required=False)
+    platform = serializers.CharField(required=False)
+    event_id = serializers.CharField(required=True)
+    environment = serializers.CharField(required=False)
+    release = serializers.CharField(required=False)
+    sdk = serializers.JSONField(required=False)
+    user = serializers.JSONField(required=False)
+    request = serializers.JSONField(required=False)
+    message = serializers.CharField(required=False)
```

Comment:
Can we align this with https://github.com/getsentry/sentry/pull/55566/files#diff-d147cb56cee48cbb3e14f6627c992032dd265c8b4c0ad97ac448314125a9367bR153-R185